scripts/babylon-integration/register-op-consumer-fp.sh

```diff
@@ -32,6 +32,9 @@ echo
 sleep 5
 
 echo "Registering OP consumer finality provider..."
-docker exec consumer-finality-provider /bin/sh \
-    -c "/bin/fpd register-finality-provider $OP_FP_EOTS_PK_HEX"
+OP_FP_REGISTRATION_TX_OUTPUT=$(docker exec consumer-finality-provider /bin/sh \
+    -c "/bin/fpd register-finality-provider $OP_FP_EOTS_PK_HEX")
+echo "OP consumer finality provider registration tx output: $OP_FP_REGISTRATION_TX_OUTPUT"
+OP_FP_REGISTRATION_TX_HASH=$(echo "$OP_FP_REGISTRATION_TX_OUTPUT" | jq -r '.tx_hash')
+echo "OP consumer finality provider registration tx hash: $OP_FP_REGISTRATION_TX_HASH"
```

Comment:
why do we need this? shouldn't it be included in the tx output above?

also if you have tested this new change, it's better to paste the logs in the test plan section so everyone can see the effect of changing this file

the current test plan doesn't provide enough details

Reply:
as above mentioned, it is like the debug msg

Reply:
the tx ID is included in `echo $OP_FP_REGISTRATION_TX_OUTPUT` 

why printing it twice?

Reply:
yes, printing the tx hash separately is indeed duplicated